app/modules/collaborations/models.py

```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+"""
+Collaborations database models
+--------------------
+"""
+
+from sqlalchemy_utils import Timestamp
+
+from app.extensions import db
+
+import uuid
+
+
+class Collaboration(db.Model, Timestamp):
+    """
+    Collaborations database model.
+    """
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+    title = db.Column(db.String(length=50), nullable=False)
```

Comment:
@naknomum could confirm, but i think this also needs a type (read or write)

Reply:
yes, there are read/edit (aka write?) in current Collaborations on wb

Reply:
Let's plan to enumerate the types of collaborations soon.  We will need to actually made three kinds of collaboration models that inherit from Collaboration: user-to-user, user-to-org, and org-to-org

Reply:
i think we only need to support user-to-user collaborations. the only way to collaborate with an org should be to join that org.